6/ansible-6.5.0.deps

```diff
@@ -0,0 +1,107 @@
+_ansible_version: 6.5.0
+_ansible_core_version: 2.13.5
+amazon.aws: 3.5.0
```

Comment:
I'm missing the `_python` entry that would be added by antsibull 0.51.0. I think it's ok to do the current release as-is, but please make sure to use the latest antsibull release for the next Ansible release :)

Reply:
@felixfontein I used the latest version originally, but CI complained about the python3.9 line, so I reverted back to 0.49.0, which was right before this.

Reply:
This change I think: https://github.com/ansible-community/antsibull/pull/448

Reply:
Ah, that's helpful to know. There seems to be a bug in there, which for some reason CI didn't complain about so far. I'll try to look at that today or tomorrow. If anyone else is interested, the error is here: https://github.com/ansible-community/ansible-build-data/actions/runs/3236677877/jobs/5302816585#step:10:98